src-tauri/src/main.rs

```diff
@@ -202,7 +202,12 @@ async fn main() -> Result<()> {
 
             Ok(())
         })
-        .plugin(tauri_plugin_window_state::Builder::default().build())
+        .plugin(
+            tauri_plugin_window_state::Builder::new()
+                .skip_initial_state(METER_WINDOW_LABEL)
+                .skip_initial_state(LOGS_WINDOW_LABEL)
```

Comment:
This breaks the previous behavior of bringing `logs_window` to the front, even though we are manually restoring its state above 🤔
https://github.com/snoww/loa-logs/blob/f241f24c36d586d352d9e777ae56cea1a83e62e2/src-tauri/src/main.rs#L129